src/components/postComments/container/postComments.tsx

```diff
@@ -106,7 +119,7 @@ const PostComments = forwardRef(
           content,
         },
         key: content.permlink,
-      });
+      } as never);
```

Comment:
what's the benefit of `as never`, haven't seen or used this before personally, just curious.

Reply:
it is not the right way to use never, never essentially points towards a no return type...here however using it temporarily to remove ugly red lines from code, right way would be to cast every `useNavigation()` with right type.